content/id/docs/concepts/storage/volume-snapshot-classes.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Volume Snapshot Class
+content_template: templates/concept
+weight: 30
+---
+
+{{% capture overview %}}
+
+Laman ini menjelaskan tentang konsep VolumeSnapshotClass pada Kubernetes. Sebelum melanjutkan,
+sangat disarankan untuk membaca [_snapshot_ volume](/docs/concepts/storage/volume-snapshots/)
+dan [kelas penyimpanan (_storage class_)](/docs/concepts/storage/storage-classes) terlebih dahulu.
+
+{{% /capture %}}
+
+
+{{% capture body %}}
+
+## Pengenalan
+
+Seperti halnya StorageClass yang menyediakan cara untuk para admin mendefinisikan
+"kelas" penyimpanan yang mereka tawarkan saat proses penyediaan sebuah volume, VolumeSnapshotClass
+menyediakan cara untuk mendefinisikan "kelas" penyimpanan saat menyediakan _snapshot_ volume.
+
+## Sumber Daya VolumeSnapshotClass
+
+Masing-masing VolumeSnapshotClass terdiri dari _field_ `snapshotter` dan `parameters`,
```

Comment:
VolumeSnapshotClass -> `VolumeSnapshotClass`

Reply:
the original page styling is incorrect, for resource name it should be camel case according to https://kubernetes.io/docs/contribute/style/style-guide/#use-camel-case-for-api-objects